response_operations_ui/templates/reporting-unit.html

```diff
@@ -170,14 +170,23 @@ <h4 class="neptune">Respondents</h4>
               </dl>
             </td>
             <td name="tbl-respondent-status" class="table--cell">
-              {{ respondent.status.title() }}
+              {% if respondent.status  == 'SUSPENDED' %} Locked {% else %} {{ respondent.status.title() }} {% endif %}
+              <div>
+{#                {% if respondent.status == 'ACTIVE' %}#}
+{#                <a href="{{ url_for('reporting_unit_bp.confirm_change_respondent_status', ru_ref=ru.sampleUnitRef, respondent_first_name=respondent.firstName,#}
+{#                                    respondent_last_name=respondent.lastName, email_address=respondent.emailAddress, change_flag='LOCKED')}}"#}
+{#                   id="change-respondent-status">Lock</a>#}
```

Comment:
What are the `{#` and `#}` for?

Reply:
So this is the ability to `lock the account` with a link but we dont want to implement yet, so I have commented out the html for now